Adding some changes to answer common questions (#7060)
cloudflare/cloudflare-docs: content/cloudflare-one/applications/configure-apps/dash-sso-apps.md

By adding a **Dashboard SSO application** to your Cloudflare Zero Trust account, you can enforce single sign-on (SSO) to the Cloudflare dashboard with the identity provider (IdP) of your choice.

> Once you have configured SSO, every user that wants to login with Dashboard SSO will need to exist as a member in the Cloudflare account and the IdP. If there is no Cloudflare member account for this user, the request will not forward the authentication phase to the Identity Provider. Refer to [Managing Cloudflare account access](/fundamentals/account-and-billing/account-setup/manage-account-members/) for information on adding users to your Cloudflare account.

Reviewer: ```suggestion
Once you have configured SSO, every user that wants to login with Dashboard SSO will need to exist as a member in your Cloudflare account and IdP. If there is no Cloudflare member account for this user, the request will not forward the authentication phase to the Identity Provider. Refer to [Managing Cloudflare account access](/fundamentals/account-and-billing/account-setup/manage-account-members/) for information on adding users to your Cloudflare account.
```

### Step 3 — Contact your account team

> Ask your account team to approve and create your SSO domain. The SSO domain can be configured for multiple email domains associated with the members in your Cloudflare Account. SSO will not be enforced on members in which the email address was not specificied. For instance, SSO is configured for email domains ending in @example.com for Account A. Therefore member with @test.com belonging to the same account, Account A, would not be prompted with the "Log in with SSO" otion and would enter their username and password. 

Reviewer: ```suggestion
Ask your account team to approve and create your SSO domain. The SSO domain can be configured for multiple email domains associated with the members in your Cloudflare account. If a member's email domain is not included in your SSO configuration, they would not see the "Log in with SSO" option and must enter their username and password.
```
